### DISCUSSION TITLE
为什么我的父子传值 接受不到父组件传值

### DISCUSSION THREAD

**Discussion by dreamsBig** [NervJS/taro#10156] [Q & A]
[项目目录.md](https://github.com/NervJS/taro/files/7080774/default.md)

**Comment by dreamsBig** [comment]
这个社区太少人 刚刚去小程序社区问 被赶了

**Comment by wangfeijing** [comment]
是不是更新的值都不走setup,watch都没用.曲线救国吧,.用父传到store,子更新从store拿计算属性值,如果只是页面组建的话数据更新未渲染加key,不知道和你的问题符不符合
